sql/ddl_internal.sql

```diff
@@ -259,26 +272,39 @@ $BODY$
 $BODY$;
 
 -- Drop chunks older than the given timestamp. If a hypertable name is given,
--- drop only chunks associated with this table.
+-- drop only chunks associated with this table. Any of the three arguments
+-- can be NULL meaning "all values".
 CREATE OR REPLACE FUNCTION _timescaledb_internal.drop_chunks_older_than(
```

Comment:
Nit: I would actually make this an "internal" function that would be discouraged to call directly,  and then have two functions: `drop_chunks()` and `drop_chunks_older_than()` that  don't accept NULL arguments but internally calls the "internal" function.

Reply:
I am confused. This is an internal function by virtue of it being in the _timescaledb_internal schema. What do you mean?